README.md

```diff
@@ -53,10 +55,11 @@ custom:
     logLevel: "info"
     enableXrayTracing: false
     enableDDTracing: true
+    enableAPIGatewayLogs: true
```

Comment:
It seems that this is the only place where the `enableAPIGatewayLogs` parameter is. It seems like the parameters you added were [these](https://github.com/DataDog/serverless-plugin-datadog/pull/127/files?file-filters%5B%5D=.md&file-filters%5B%5D=.sh&file-filters%5B%5D=.ts&file-filters%5B%5D=.yml#diff-00ee6a506727e42b3fb03361b8216c805ede24dc3985d237de852a73bda59f30R53) ones (`subscribeToApiGatewayLogs`, `subscribeToHttpApiLogs`, and `subscribeToWebsocketLogs`). 

Basically I am wondering is `enableAPIGatewayLogs` a dead parameter? I dont see any addition to the plugin code that supports this parameter. 

Maybe these 3 (`subscribeToApiGatewayLogs`, `subscribeToHttpApiLogs`, and `subscribeToWebsocketLogs`) should be documented in the readme?